roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -1624,7 +1683,54 @@ RooFitResult* RooAbsPdf::fitTo(RooAbsData& data, const RooLinkedList& cmdList)
 	// Evaluate errors with Hesse
 	m.hesse() ;
       }
-      
+
+      //asymptotically correct approach
```

Comment:
Why does this show up twice? Is the code identical, and only parameters change? If yes, can this be extracted into a function such that the code has to be checked and maintained only once?

Reply:
This only shows up twice because I followed the exact procedure/structure that is done for the SumW2Errors. If I understand correctly the two code blocks are for the two possible different minimizers, but these should not affect the code (on the other hand I think the same should be the case for the SumW2Errors). So I agree that this could probably be improved. However, as the whole thing is put in preprocessor directives I did not want to touch it/change the structure too much.

Reply:
Oh, I didn't see that it's inside ifdefs. Ok, thanks for clarifying that.